linux_os/guide/services/ssh/ssh_server/var_sshd_set_maxstartups.var

```diff
@@ -0,0 +1,14 @@
+documentation_complete: true
+
+title: 'SSH MaxStartups setting'
+
+description: 'Configure parameters for maximum concurrent unauthenticatd connections to the SSH daemon.'
+
+type: string
+
+operator: equals
+
+interactive: true
+
+options:
+    '10:30:60': '10:30:60'
```

Comment:
You can keep the default entry here with the same value.

Reply:
Re-added, thanks.